src/core/dbus-manager.c

```diff
@@ -2184,10 +2185,19 @@ static int reply_unit_file_changes_and_free(
         return r;
 }
 
+static int mac_unit_callback_check(const char *unit_name, void *userdata) {
+        MacUnitCallbackUserdata *ud = userdata;
+
+        assert(unit_name);
+        assert(ud);
+
+        return 0;
+}
```

Comment:
maybe split commit in two: first add the infra to install.c but pass NULL/NULL everywhere, then change the dbus manager code to actually pass something better than NULL/NULL.